test/e2e/performanceprofile/functests/utils/consts.go

```diff
@@ -101,6 +101,8 @@ const (
 	NamespaceMachineConfigOperator = "openshift-machine-config-operator"
 	// NamespaceTesting contains the name of the testing namespace
 	NamespaceTesting = "performance-addon-operators-testing"
+	// DeploymentName contains the name of the deployment
+	DeploymentName = "test-deployment"
```

Comment:
Let's just move this constant to test/e2e/performanceprofile/functests/11_mixedcpus/mixedcpus.go and use it there and we're good to go

I should noticed the location of the constant earlier - my bad

Reply:
Done